src/style.css

```diff
@@ -1,3 +1,71 @@
-body {
+:root {
+  --blue-color: #00acdc;
+}
+
+* {
```

Comment:
Please don't use `*` selectors. This might have a negative impact to performance, since browser engine has to apply such styles to every single item in HTML. That's why it's prohibited in our code style guide.
Reset of margin and padding would be sufficient for `body` only.